Add basics aa1 to master

Aichnerc opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no application.

**What is the new behavior (if this is a feature change)?**
- It is now possible to see the status of websites built and/or maintained by us.

**Other information**:
- The version has been set to `v1.0.0`, which marks the beginning of this application.

laviniabergetta committed:
Remove unused pages

Removed LoginPage, ProfilPage & MessagePage.

laviniabergetta committed:
Implement status list

Added status list and status list items.

Aichnerc committed:
Update CNAME

The cname has been set to the cloud.

Aichnerc committed:
Remove Navbar

The navigation bar has been removed.

Aichnerc committed:
Update logo

The logo has been updated.

Aichnerc committed:
Improve functionality and design

The system has been greatly improved.

Aichnerc committed:
Improve README.md

The content has been improved to fit the theme.

Aichnerc committed:
Improve status image

The quality has been improved.

Aichnerc committed:
Update README.md

Some things have been improved.

pinterid commented on README.md:
```diff
@@ -30,6 +29,7 @@
 
 - [Table of contents](#table-of-contents)
 - [Quick start](#quick-start)
+  - [This project requires the MDB React Pro license and its library](#this-project-requires-the-mdb-react-pro-license-and-its-library)
```
```suggestion
- [This project requires the MDB React Pro license and its library](#this-project-requires-the-mdb-react-pro-license-and-its-library)
```

pinterid commented on src/components/molecules/StatusItem/index.jsx:
```diff
@@ -0,0 +1,268 @@
+
+//#region > URLS
+const PROXY_URL = "https://cors-anywhere.herokuapp.com/";
+////#endregion
```
```suggestion
//#endregion

```

pinterid commented on src/Routes.js:
```diff
@@ -8,25 +8,13 @@ import { Route, Switch } from "react-router-dom";
-  ProfilePage,
-} from "./components/pages";
+import { HomePage } from "./components/pages";
 
```
This files has no regions.

pinterid commented on src/components/molecules/StatusItem/index.jsx:
```diff
@@ -0,0 +1,268 @@
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+
```
```suggestion
```

pinterid commented on src/components/molecules/StatusItem/index.jsx:
```diff
@@ -0,0 +1,268 @@
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import { MDBListGroupItem, MDBIcon } from "mdbreact";
+
```
```suggestion
```

pinterid commented on src/components/molecules/StatusItem/index.jsx:
```diff
@@ -0,0 +1,268 @@
+import axios from "axios";
+//#endregion
+
+//#region > URLS
```
```suggestion
//#region > Constant Variables

```

pinterid commented on src/components/molecules/StatusItem/index.jsx:
```diff
@@ -0,0 +1,268 @@
+//#region > Components
+class StatusItem extends React.Component {
+  constructor(props) {
+    super(props);
```
```suggestion
    super(props);
    
```

pinterid commented on src/components/molecules/StatusItem/index.jsx:
```diff
@@ -0,0 +1,268 @@
+            const status = response.status;
+            const receiveDate = new Date().getTime();
+
+            console.log(response);
```
Is this required?

pinterid reviewed (changes requested):
Looks good after a few changes.

Aichnerc committed:
Improve code quality

Some changes have been made to code quality and documentation due to requests by @pinterid.

Aichnerc committed:
Update README.md

A unused headline has been removed from the table of contents.

Aichnerc committed:
Remove automatic refetching

There were too many requests it seems.

Aichnerc committed:
Improve code stability

The cors have been changed.

Theralun reviewed (approved):
Lgtm.

pinterid reviewed (approved):
Looks good.

Aichnerc committed:
Remove unused code

Unused code has been removed.
